addons/event_booth_sale/views/event_booth_views.xml

```diff
@@ -41,13 +41,13 @@
     </record>
 
     <record id="event_booth_view_tree" model="ir.ui.view">
-        <field name="name">event.booth.view.tree.inherit.sale</field>
+        <field name="name">event.booth.view.tree.from.event.inherit.sale</field>
```

Comment:
Name still includes "from.event", to be cleaned?

Reply:
Thanks, changed.